client/src/client/es/client/client.ts

```diff
@@ -95,7 +95,7 @@ export function createApiClient(
       requestInit.body = JSON.stringify(body);
     }
 
-    if (method === 'GET') {
+    if (method === 'GET' || (method === 'DELETE' && params)) {
```

Comment:
```suggestion
['GET', 'DELETE'].includes(method)
```
Perhaps something along this?